app/views/shared/_personal_key.html.erb

```diff
@@ -1,51 +1,55 @@
-<%= render PageHeadingComponent.new.with_content(t('headings.personal_key')) %>
-<p>
-  <%= t('instructions.personal_key.info') %>
-</p>
+<%= render PageHeadingComponent.new.with_content(t('forms.personal_key_partial.header')) %>
 <div class="full-width-box margin-y-5">
   <%= render 'partials/personal_key/key', code: code %>
 </div>
-<%= render ButtonComponent.new(
-      action: ->(**tag_options, &block) do
-        link_to(
-          "data:text/plain;charset=utf-8,#{CGI.escape(code)}",
-          download: 'personal_key.txt',
-          **tag_options,
-          &block
-        )
-      end,
-      icon: :file_download,
-      outline: true,
-      class: 'margin-right-2 margin-bottom-2 tablet:margin-bottom-0',
-    ).with_content(t('forms.backup_code.download')) %>
-<%= render PrintButtonComponent.new(
-      icon: :print,
-      outline: true,
-      type: :button,
-      class: 'margin-right-2 margin-bottom-2 tablet:margin-bottom-0',
-    ) %>
-<%= render ClipboardButtonComponent.new(
-      clipboard_text: code,
-      outline: true,
-      class: 'margin-bottom-2 tablet:margin-bottom-0',
-    ) %>
-<div class="margin-y-5 clearfix">
-  <%= image_tag(
-        asset_url('alert/icon-lock-alert-important.svg'),
-        alt: '',
-        size: '80',
-        class: 'float-left margin-right-2',
-      ) %>
-  <p class="margin-bottom-0">
-    <strong><%= t('instructions.personal_key.email_title') %></strong>
-  </p>
-  <p><%= t('instructions.personal_key.email_body') %></p>
+
+<br />
+
+<div>
+  <%= render AccordionComponent.new do |c| %>
+    <% c.header { t('forms.personal_key_partial.explanation.header') } %>
+    <% t('forms.personal_key_partial.explanation.text').each do |paragraph| %>
+      <p><%= paragraph %></p>
+    <% end %>
+  <% end %>
 </div>
-<%= button_to(
-      t('forms.buttons.continue'),
-      update_path,
-      class: 'display-block usa-button usa-button--big usa-button--wide personal-key-continue margin-top-5',
-      'data-toggle': FeatureManagement.idv_personal_key_confirmation_enabled? ? 'modal' : 'skip',
-    ) %>
+
+<%= simple_form_for('', url: update_path) do |f| %>
+  <% @acceptance_prompt = capture do %>
+    <p>
+      <b>
+        <%= t('forms.personal_key_partial.acknowledgement.header') %>
+      </b>
+    </p>
+    <p>
+      <%= t('forms.personal_key_partial.acknowledgement.text') %>
+      <ul>
+        <% t('forms.personal_key_partial.acknowledgement.bullets').each do |bullet| %>
+          <li><%= bullet %></li>
+        <% end %>
+      </ul>
+    </p>
+    <p>
+      <%= t('forms.personal_key_partial.acknowledgement.instructions') %>
+    </p>
+  <% end %>
+
+  <%= render ValidatedFieldComponent.new(
+        form: f,
+        name: :acknowledgment,
+        as: :boolean,
+        label: @acceptance_prompt,
```

Comment:
I'm a bit concerned about the accessibility implications of this label being so long, since a user has to read through quite a bit of text before they can understand what the label is controlling (the checkbox), and this may pose difficulty for those relying on assistive technology to navigate the page.

Also, because `<label>` is only meant to contain inline content, this produces invalid HTML, since we're rendering multiple paragraphs and lists within the label.

Personally I'd prefer if we'd separate the explanation text from the checkbox acknowledgment similar to what we do earlier in the "How verifying your identity works", but I assume that'd ultimately be up to the UX folks on your team.

Reply:
Changed as per slack discussion.